todo/src/components/todo.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {addToDo, removeToDo} from '../actions/action'
+//import ReactDOM from 'react-dom';
+import {connect} from 'react-redux';
+import './todo.css'
+import ls from 'local-storage';
+
```

Comment:
good job on your todo logic (and remember to add the e,preventDefault() when appropriate).